<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	background-color:rgb(227, 227, 227);
	border:1px solid;
	border-radius:8px;
	font: 14pt &quot;Yagut&quot;;
	padding:5px;
}

QLabel{
	font: 14pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/postman.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>821</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 52), stop:1 rgba(0, 0, 0, 56))</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
	border:1px solid rgb(227, 227, 227);
	border-radius:8px;
}

QPushButton:hover{
background-color:rgb(51, 167, 82);
border:1px solid rgb(227, 227, 227);
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/Gmail-logo-500x281.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/bubble-160786.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="text_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
color:#000;</string>
   </property>
   <property name="text">
    <string>Hello, i'm google postman.</string>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
color:#000;</string>
   </property>
   <property name="text">
    <string>if you have any mail to send. let me know</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Yagut&quot;;
	color:rgb(227, 227, 227);
	border:1px solid rgb(227, 227, 227);
	border-radius:8px;
}

QPushButton:hover{
background-color:rgb(233, 66, 52);
border:1px solid rgb(227, 227, 227);
}</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>37</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
